Assets/HoloToolkit-Examples/Utilities/Scripts/CheckApiTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CheckApiTest : MonoBehaviour
+{
+    [SerializeField]
+    private Text text;
+
+    public void CheckV5Api()
```

Comment:
Pressing the buttons with a motion controller doesn't seem to call this code (voice "select" works). Is that something that'll be fixed with your uGUI fixes?

Reply:
That is correct. Will be fixed by the uGUI raycast PR